client/src/tests/questions.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow, mount, render } from 'enzyme';
+import Questions from '../Components/Questions.jsx';
+
+// These tests are not passing, I may have to pass in mock data for the questions props?
+
+describe('<Questions />', () => {
+  test('it should call a function when the previous page button is clicked', () => {
+    const mockSubmitHandler = jest.fn();
+    const wrapper = shallow(<Questions questions={{}}/>);
+    wrapper.instance().nextPage = mockSubmitHandler;
```

Comment:
looks like you may have the nextPage and prevPage properties switched - this test is referencing nextPage, and the following test is referencing prevPage